src/fitnesse/testsystems/slim/tables/SlimTable.java

```diff
@@ -200,7 +203,28 @@ public void setCustomComparatorRegistry(CustomComparatorRegistry customComparato
     this.customComparatorRegistry = customComparatorRegistry;
   }
 
-  /** SlimExpectation base class for row based expectations. */
+  private String getSlimExpressionResult(String variableNameWithDollar) {
+    SlimExpressionEvaluator evaluator = new SlimExpressionEvaluator();
+    String expr = variableNameWithDollar.substring(2, variableNameWithDollar.length() - 1);
+    String variableName = expr.split("\\.")[0];
+
+    if (getSymbol(variableName) != null) {
+      Map<String, MethodExecutionResult> symbols = new HashMap<>();
+      symbols.put(variableName, new MethodExecutionResult(getSymbol(variableName), Object.class));
```

Comment:
Shouldn't all symbols from the test's context be made accessible to the evaluator (the expression might reference more than one variable, and might not start with a variable's name). I mean the expression could be `1.5 * myHashMap.size() + myHashMap2.key2.length()`. 
Or am I misinterpreting something?